docs/modules/ROOT/pages/tmail-backend/configure/twake-workplace.adoc

```diff
@@ -52,4 +52,8 @@ To enable SaaS integration module, you need to load the corresponding extension
 com.linagora.tmail.james.app.MemorySaaSModule
 com.linagora.tmail.james.app.DistributedSaaSModule
 com.linagora.tmail.james.app.PostgresSaaSModule
-----
\ No newline at end of file
+----
+
+=== SenderIsPayingUser matcher
```

Comment:
Sorry to be picky but a second though (with fresh eyes) on this I think we would be better served by having the following matchers:

`HasAnySaaSPlan`
`HasSaasPlan=standard,premium` (coma separated list)

That way we could potentially write in the conf any logic we want and NOT a single use case.

Thought @quantranhong1999 @Arsnael ?